### ISSUE TITLE
Error messages incorporation

### ISSUE THREAD

**Issue by NadiaHuisjes** [duderstadt-lab/mars-fx#5]
Hi Karl,

Can you maybe add error messages in the following cases:
- When you try to export the bdv video whilst not having the video viewer open
- When you do not set the parameters for the bdv viewer correctly, or if you did not couple the xml files at all
- When you try to open a virtual storage with the normal open command
- When you try to open a normal archive with the virtual storage command 

And maybe also
- When you try to remove a non-existing parameter or tag

**Comment by NadiaHuisjes** [comment]
Another idea for an error message:
* When you want to execute the 'export video' command and the BDV video is not running already an error should show up noting that you should run the 'show video' command first before running this one.

**Comment by NadiaHuisjes** [comment]
And a proposed warning message:
* -	The ‘Filter Table’ function generates a new table based on the settings specified as input. I was wondering whether we should include some sort of a warning when out of boundaries values are getting discarded? Feels so drastic to lose them immediately without a warning.

**Comment by karlduderstadt** [comment]
Great idea, thanks @NadiaHuisjes for keeping track of all the errors we need to response to. I have added a method in mars-fx to show error messages with a Rover Icon and so far I have added the error:
*   When you try to export the bdv video whilst not having the video viewer open
This was added in @9918e002366362a11c02691667e0dc78d0ad6fb1. Forgot to mention the issue number, so I just added the hard reference here.
<img width="563" alt="Screenshot 2020-09-14 at 21 20 18" src="https://user-images.githubusercontent.com/17951396/93129021-ac25a480-f6d0-11ea-82d2-ca1ac5f53e6f.png">

**Comment by mjosch** [comment]
Hi,

Concerning setting up the BDV, in case a drift column was provided I'd suggest to cross-check if the drift column is present in the metadata. Happens easily to have a typo in there and it's hard to notice without receiving an error message.

What do you think?

**Comment by karlduderstadt** [comment]
Good idea, I guess this would be during the creation of a new BDV entry in the metadata record that the error would be thrown?

**Comment by mjosch** [comment]
By default it's created without affine transform parameters and drift columns, so those are added after creation. Maybe check once the viewer is initialized ("show or export video")?

**Comment by NadiaHuisjes** [comment]
Another Error message idea:
* When opening the 'Peak Tracker' in case no image/video is selected we get the NullPointer error. Maybe also replace this with a more specific 'Please open an image first before running this command' error or such

**Comment by karlduderstadt** [comment]
Ahh yes, Nadia the NullPointer error was happening in the initialize() method before the dialog shows up. The command was doing some work with the imageDisplay to build the dialog. But if not imageDisplay is there then there is an NullPointerException.

Now I just stop running the initialize() method if the imageDisplay does not exists and actually scijava automatically makes the error message since a required input is not there... 

I made this change for the other image processing commands as well.
